Content.Shared/GameObjects/Components/Items/ClothingComponentState.cs

```diff
@@ -0,0 +1,17 @@
+﻿using SS14.Shared.GameObjects;
+using SS14.Shared.Serialization;
+using System;
+
+namespace Content.Shared.GameObjects.Components.Items
+{
+    [Serializable, NetSerializable]
+    public class ClothingComponentState : ComponentState
```

Comment:
Make this a subclass of `ItemComponentState` so the item equipped prefix gets carried through too.

Reply:
No, then the call to the base constructor gets jangled up by trying to throw it into the constructor for ItemComponent.

Reply:
Then make it take two params, one for the item equip prefix and one for the clothing equip prefix.

Either way the clothing component state *replaces* the item component state, so without doing this you won't be able to replicate the item equip prefix anymore for clothing items.

Reply:
Duplicated the field from ItemComponentState into ClothingComponentState.